crates/rooch/src/commands/server/commands/clean.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use clap::Parser;
+use rooch_config::store_config::StoreConfig;
+use rooch_types::error::{RoochError, RoochResult};
+use std::{fs, path::Path};
+
+/// Clean the Rooch server storage
+#[derive(Debug, Parser)]
+pub struct CleanCommand;
+
+impl CleanCommand {
+    pub async fn execute(self) -> RoochResult<()> {
+        let rooch_store = StoreConfig::get_rooch_store_dir();
+        let moveos_store = StoreConfig::get_moveos_store_dir();
+
+        self.remove_store_dir(&rooch_store, "Rooch")?;
+        self.remove_store_dir(&moveos_store, "MoveOS")?;
+
+        println!("Rooch server storage successfully cleaned");
```

Comment:
Do we need to remove the Keystore file in the clean command? @wow-sven

Reply:
that doesn't remove the keystore @jolestar,  It's the dbpath

Reply:
the keystore is stored in `~/.rooch/rooch_config` while the code removes files under `~/.rooch/roochdb` @jolestar

Reply:
Yes. I want to ask about the goal of clean command. Sometimes, the Keystore serialize format changes also break the CLI, so should we support cleaning the Keystore?

Reply:
the goal of the clean command as far as I can tell is to remove the server db storage i.e rooch db store and moveos db store, in some cases that command might be needed.

regarding the keystore, cleaning the keystore doesn't make make sense and very risky because the users private keys are stored there. so, cleaning it means the user will loose their keys forever

Reply:
Ok, we need to care about the keystore file format.